pages/_app.js

```diff
@@ -6,20 +6,47 @@ import { vocs } from "../lib/db";
 
 import Layout from "../components/Layout";
 
+const initialSettings = [
+  {
+    name: "numCardsToPick",
+    label: "Cards per game:",
+    value: 5,
+  },
+  {
+    name: "hitsToWin",
+    label: "Iterations to win:",
+    value: 3,
+  },
+];
 
 export default function App({ Component, pageProps }) {
   const [cards, setCards] = useLocalStorageState("cards", {
     defaultValue: vocs,
   });
+  const [settings, setSettings] = useLocalStorageState("voc_settings", {
+    defaultValue: initialSettings,
+  });
+  function handleChangeSettings(name, newValue) {
```

Comment:
Looks and feels good to have the handler function in the same place as the state.